db/migrate/20150811020817_create_transactions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTransactions < ActiveRecord::Migration
+  def change
+    create_table :transactions do |t|
+      t.integer :amount, default: 0, null: false, index: true
```

Comment:
Maybe the dollar amount would be better stored as decimal? [Postgres doc](http://www.postgresql.org/docs/9.1/static/datatype-numeric.html) recommends it for storing monetary amounts. That way we don't have to manage the conversion from dollars to cents.